pkg/cmd/attestation/verification/extensions.go

```diff
@@ -0,0 +1,27 @@
+package verification
+
+import (
+	"fmt"
+	"strings"
+)
+
+func VerifyCertExtensions(results []*AttestationProcessingResult, owner string, repo string) error {
+	for _, attestation := range results {
+		if owner != "" {
+			expectedSourceRepositoryOwnerURI := fmt.Sprintf("https://github.com/%s", owner)
```

Comment:
with proxima, this prefix (`https://github.com`) is going to change depending on the tenant. this should not be hardcode here; maybe the prefix is passed in as an option, or is set to a default if the option is `""`?

Reply:
added a todo for future task for proxima